python/py-machfs/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=Portfile:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-machfs
+version             1.2.4
+
+categories          devel
+license             MIT
+platforms           darwin
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         a library for creating and inspecting HFS-format disk images
+long_description    This is {*}${description}. Mac-specific concepts like resource forks \
+                    and type/creator codes are first-class citizens.
+
+homepage            https://github.com/elliotnunn/machfs
+
+python.versions     38
+
+if {${name} ne ${subport}} {
+    depends_build       port:py${python.version}-setuptools
+    depends_lib         port:py${python.version}-macresources
```

Comment:
You should append to depends_build and depends_lib so as not to overwrite dependencies that might be set by portgroups.